ui/components/app/hold-to-reveal-button/index.scss

```diff
@@ -0,0 +1,164 @@
+// Variables
+$circle-radius: 14px;
+$circle-diameter: $circle-radius * 2;
+// Circumference ~ (2*PI*R). We reduced the number a little to create a snappier interaction
+$circle-circumference: 82;
+$circle-stroke-width: 2px;
+
+// Keyframes
+@keyframes collapse {
+  from {
+    transform: scale(1);
+  }
+
+  to {
+    transform: scale(0);
+  }
+}
+
+@keyframes expand {
+  from {
+    transform: scale(0);
+  }
+
+  to {
+    transform: scale(1);
+  }
+}
+
+@keyframes fadeIn {
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+}
+
+// Shared styles
+%absolute-fill {
```

Comment:
Can we scope these mixins within a `.hold-to-reveal-button` class? That way they don't get created globally.

Reply:
Fixed in: 2afe9c604ab6d1e79ede7a1d02693f1a34d96e3c